### DISCUSSION TITLE
Complete example of popover usage

### DISCUSSION THREAD

**Discussion by mayur-ekbote** [twbs/bootstrap#33744] [Q&A]
Earlier, with jQuery, the use of popver was straightforward. 
With jQuery no longer used, how does one use Popovers? Specifically if I am not using JS (pure HTML / Blazor)
What to do with the script provided in the document? Eg.
var popoverTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="popover"]'))
var popoverList = popoverTriggerList.map(function (popoverTriggerEl) {
  return new bootstrap.Popover(popoverTriggerEl)
})

A complete example would be of great help.

**Comment by hax18** [comment]
I'm not sure if i really understand ur question, but u should already konw that popover in **Bs5** still rely on a 3rd party library "**Popper.js**" !

And this piece of code will help to **Initialize**  & **Enable** Popover on ur website.
```
 var popoverTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="popover"]'))
 var popoverList = popoverTriggerList.map(function (popoverTriggerEl) {
 return new bootstrap.Popover(popoverTriggerEl)
 })
```

**Reply by mayur-ekbote** [reply]
Yes. The documentation says that either I include the popper or use the js bundle.
The question is, were do I put the above code? Earlier, you could do a $(document). with jQuery.  What to do now?

**Reply by hax18** [reply]
**`Popper must come first`**. Then **`Bootstrap`**  and finally u have to put the code or the file contains that peice of code **_`under`_** the **Poper.js** 

**Example:**

> Poper Js

`<script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.1/dist/umd/popper.min.js"></script>`

> Bootstrap

`<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.min.js"></script>`

> Js Code

`<script src="code.js"></script>`

**Reply by mayur-ekbote** [reply]
Thanks. I was actually looking for this (I use blazor so I have no idea how to make js work)
Keeping it here in case someone comes looking for it:


```
<body>
....
....
....

    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js" integrity="sha384- JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf" crossorigin="anonymous">
    </script>

    <script>

      document.querySelectorAll('[data-bs-toggle="popover"]')
        .forEach(function (popover) {
        new bootstrap.Popover(popover)
        })
    </script>
</body>
```

**Reply by hax18** [reply]
You are welcome
